x-pack/plugins/lists/server/routes/internal/create_exceptions_list_route.ts

```diff
@@ -20,7 +20,9 @@ export const internalCreateExceptionListRoute = (router: ListsPluginRouter): voi
   router.post(
     {
       options: {
-        tags: ['access:lists-all'],
+        // Access control is set to `read` on purpose, as this route is internal and meant to
+        // ensure we have lists created (if not already) in order for searches to work.
+        tags: ['access:lists-read'],
```

Comment:
So this is needed in order to ensure that the lists for Endpoint artifacts are created for users that only have `read` RBAC. What this does, however, is now enables anyone with `read` to create lists, something that we probably don't want. 

I'm pretty sure this API is only currently used by us (Endpoint), so I'm thinking that we should add a check on this API route to ensure that the lists that it attempt to create are only those that support Endpoint artifacts.

@dasansol92 , @MadameSheema ,
Your thoughts here?

Reply:
I think adding the check to ensure calls are from endpoint artifacts makes sense

Reply:
Thanks for the feedback. I'm going to add that in with the following condition:

1. If user does NOT have `all` permissions, then validate that the request body is for an Endpoint artifact list only
2. If user has `all`, then allow it as we do today

Reply:
Ok. After looking into this again, I found that the API's schema already validates that the only list IDs allowed are those for Endpoint artifacts, so no additional changes needed